suite-native/device/src/screens/DeviceInfoModalScreen.tsx

```diff
@@ -39,15 +39,18 @@ import { useOpenLink } from '@suite-native/link';
 import { HowToUpdateBottomSheet } from '../components/HowToUpdateBottomSheet';
 
 const deviceImageMap = {
-    [DeviceModelInternal.T1B1]: require('../assets/t1.png'),
-    [DeviceModelInternal.T2T1]: require('../assets/tt.png'),
-    [DeviceModelInternal.T2B1]: require('../assets/ts3.png'),
+    [DeviceModelInternal.T1B1]: require('../assets/t1b1.png'),
+    [DeviceModelInternal.T2T1]: require('../assets/t2t1.png'),
+    [DeviceModelInternal.T2B1]: require('../assets/t2b1.png'),
+    [DeviceModelInternal.T3T1]: require('../assets/t3t1.png'),
 } as const satisfies Record<DeviceModelInternal, string>;
 
+// TODO: there is device.name from device in connect. why not use it?
 const deviceModelNameMap = {
     [DeviceModelInternal.T1B1]: 'Trezor Model One',
     [DeviceModelInternal.T2T1]: 'Trezor Model T',
     [DeviceModelInternal.T2B1]: 'Trezor Safe 3',
+    [DeviceModelInternal.T3T1]: 'Trezor Safe 5',
 } as const satisfies Record<DeviceModelInternal, string>;
```

Comment:
we can replace satisfly with just type-definition

Reply:
removed completely